tcms/report/views.py

```diff
@@ -81,73 +82,83 @@ def overview(request, product_id, template_name='report/overview.html'):
 
 class ProductVersionReport(TemplateView, ProductVersionReportData):
     template_name = 'report/version.html'
+    product = None
+    selected_version = None
 
-    def get(self, request, product_id):
+    def get(self, request, *args, **kwargs):
         try:
-            self.product = Product.objects.only('name').get(pk=product_id)
-
+            self.product = Product.objects.only('name').get(pk=args[0])
```

Comment:
Does `product_id = kwargs.get('product_id')` work ?

Reply:
Do you mean you want me to replace
`self.product = Product.objects.only('name').get(pk=args[0])`
with
`product_id = kwargs.get('product_id')`?

Reply:
Yes.

`args[0]` doesn't really say what it is all about. `kwargs.get('product_id')` on the other hand is much more understandable.